x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/engine_router.tsx

```diff
@@ -98,6 +100,12 @@ export const EngineRouter: React.FC = () => {
           <div data-test-subj="AnalyticsTODO">Just testing right now</div>
         </Route>
       )}
+      <Route path={ENGINE_PATH + ENGINE_DOCUMENT_DETAIL_PATH}>
+        <DocumentDetail engineBreadcrumb={engineBreadcrumb} />
+      </Route>
+      <Route path={ENGINE_PATH + ENGINE_DOCUMENTS_PATH}>
+        <Documents engineBreadcrumb={engineBreadcrumb} />
+      </Route>
```

Comment:
One potential bonus to a DocumentsRouter as well is that you'd be passing down `engineBreadcrumb` only once to the router, and the router can handle setting `<SetPageChrome/>` at the route level instead of having the views do it themselves

Reply:
Actually after seeing us use `decodeURIComponent()` on the DocumentDetails view I think that's probably more of a hassle than it's worth right now, so feel free to disregard this comment - I'll go ahead and collapse it